examples/Utils/b3Clock.cpp

```diff
@@ -143,8 +141,41 @@ unsigned long int b3Clock::getTimeMilliseconds()
 #endif
 }
 
-	/// Returns the time in us since the last call to reset or since 
-	/// the Clock was created.
+/// Gets the system time in milliseconds
+unsigned long int b3Clock::getSystemTimeMilliseconds() {
```

Comment:
Can you please remove any changes to b3Clock? I will add an option to 'reset' using absolute 0 time reference. See https://github.com/bulletphysics/bullet3/pull/1165